tool/plugin/lib/edit.dart

```diff
@@ -175,19 +135,9 @@ class EditAndroidModuleLibraryManager extends EditCommand {
           "super.init(false, indicator);", "super.init(indicator);");
       source = source.replaceAll(
           "import com.intellij.openapi.project.impl.ProjectExImpl;", "");
-      if (spec.version.startsWith("4.1")) {
-        source = source.replaceAll(
-            "import com.intellij.openapi.components.impl.stores.IProjectStore;",
-            "");
-        source = source.replaceAll("IProjectStore", "Object");
-        source = source.replaceAll(
-            "androidProject.init42", "androidProject.initPre41");
-        source = source.replaceAll("PluginManagerCore.getLoadedPlugins(), null",
-            "PluginManagerCore.getLoadedPlugins(), false");
-      }
       if (spec.version.startsWith('4.2')) {
-        source = source.replaceAll(
-            "androidProject.init42", "androidProject.init41");
+        source =
+            source.replaceAll("androidProject.init42", "androidProject.init41");
```

Comment:
We might be able to update this as well?

Reply:
Best to leave that alone for now. I'm hoping to delete it sometime soon.